code/datums/extensions/health/_defines.dm

```diff
@@ -0,0 +1,41 @@
+// Flags for damage types
+/// Generic physical damage sources - Punches, general attacks, bullets, etc
+var/global/const/DAMAGE_BRUTE     = "brute"
+/// Generic burn damage sources - Lasers, focused heat sources (Not fire, see DAMAGE_FIRE), etc
+var/global/const/DAMAGE_BURN      = "burn"
+/// Non-harming energy effects - Stun mode weapons, tasers, practice lasers, etc
+var/global/const/DAMAGE_STUN      = "stun"
+/// Electrical damage, shock rounds, less-than-lethal lasers, etc
+var/global/const/DAMAGE_SHOCK     = "shock"
+/// Strictly EMP damage, primarily used in emp_act()
+var/global/const/DAMAGE_EMP       = "emp"
+/// Strictly explosion damage, primarily used in explode_act()
+var/global/const/DAMAGE_EXPLODE   = "explode"
+/// Strictly fire damage, primarily used in fire_act()
+var/global/const/DAMAGE_FIRE      = "fire"
+/// Radiation damage
+var/global/const/DAMAGE_RADIATION = "radiation"
+/// Bio-damage. Infections, viruses, etc.
+var/global/const/DAMAGE_BIO       = "bio"
+/// Pain damage. Used for compatabiltiy with legacy code, not intended for actual health tracking use.
+var/global/const/DAMAGE_PAIN      = "pain"
+/// Toxin and poison damage.
+var/global/const/DAMAGE_TOXIN     = "toxin"
+/// Genetic damage
+var/global/const/DAMAGE_GENETIC   = "genetic"
+/// Oxyloss damage
+var/global/const/DAMAGE_OXY       = "oxy"
+/// Brain damage
+var/global/const/DAMAGE_BRAIN     = "brain"
+
+/// All damage flags
+#define DAMAGE_ALL list(DAMAGE_BRUTE, DAMAGE_BURN, DAMAGE_STUN, DAMAGE_SHOCK, DAMAGE_EMP, DAMAGE_EXPLODE, DAMAGE_FIRE, DAMAGE_RADIATION, DAMAGE_BIO, DAMAGE_PAIN, DAMAGE_TOXIN, DAMAGE_GENETIC, DAMAGE_OXY, DAMAGE_BRAIN)
```

Comment:
Why make the above consts but then keep this as a define? This is more expensive than all of the above; every time you use the define you're creating a large list.

Reply:
I wasn't able to get constant lists to work compile. I don't recall what the exact error was - Irrelevant now either way since I've removed the damage constants for now.

Reply:
Actually still relevant as I do still need this list at least for the resistance stuff to work. I'll poke on discord about the constant error.

Reply:
Pinged you on Discord, but `var/global/list/all_damage_types` would do the same job without complaining about lists being nonconstant.